docs/content/2.frontier/3.cli/1.usage.md

```diff
@@ -1,2 +1,52 @@
 # Usage
 
+Installing the CLI can be done with the following commands:
+
+```bash
+npm install -g @rdfrontier/cli
+```
+
+or
+
+```bash
+yarn global add @rdfrontier/cli
```

Comment:
Some of the code snippets in other files have the "$" prefix while others don't. Maybe consider making it consistent. Adding or removing the "$" would be fine as long as it is consistent. What do you guys prefer?

Reply:
We could have them all use '$'